### ISSUE TITLE
OrchardCore Multiple Developers Using Yessql Database

### ISSUE THREAD

**Issue by gsbanner13** [OrchardCMS/OrchardCore#7084]
Hello!
We have recently added a second dev to the team and we are both developing on a pretty straightforward OrchardCore project.  We use the built in Yessql file to hold all of our "main" site data (pages, links, etc...) and we are using Dev AzureOps TFS for Source Control.  The other day, 

I checked in some changes that I made and let the team know.  Then, when the other Dev pulled down those changes, it overwrote what he had in his Yessql file, which caused him to lose the changes he had made, which were also related to pages and links. 

What are the best practices for handling this? I.e. two developers trying to share the Yessql file?

**Comment by Skrypt** [comment]
https://github.com/OrchardCMS/OrchardCore/discussions/7052

**Comment by gsbanner13** [comment]
I'm sorry. but this does not seem to be a duplicate to me.  I found that other issue but it seems like they are talking more about multi-tenancy, which (as far as I know) is not what we are doing.  

Let me explain.  We are hosting our site on an Azure dev ops server, using TFS source control.  Inside the solution on the server, there is a 'yessql.db'  We have two developers (myself and another) working with the code via Visual Studio.  Inside of each of our solutions, we have 'yessql'db' file.  Last week, I checked in changes to my solution, which included some new pages that I had added, which (I'm assuming) are stored inside that 'yessql.db' file.  Then, when the other devloper got latest on the code, his db file was overwritten and the changes he made were gone.

What we are looking for is a solution to handle new records in the tables, not schema changes, which is what seems to be indicated by the links detailed above in this message.

Also, I noticed in my research on this topic that I don't seem to have all of the options in my admin interface.  For example, I believe that I should have an option for 'Modules' but I do not.  How do I get that?

This is all that I currently have:

![image](https://user-images.githubusercontent.com/36817126/94033983-7d25c780-fd8f-11ea-916c-697cc34bf47b.png)

**Comment by deanmarcussen** [comment]
The problem you are having is putting your SQLite database in source control and expecting that it will merge changes.

As you've already experienced that won't work.

You'll just overwrite it everytime one of you pushes. And you'll never be able to sort the merge conflicts out and have a working database at the end of it.

If you want to work with two developers on the same database you'll need to use a shared database like MS SQL.

In the linked issue I have described how our team is currently working with multiple developers on the same database - not multitenanted, just a single past paced development process.

If you're looking for Features in the menu go to _Configuration -> Features_

**Comment by gsbanner13** [comment]
Hi!  Me again.  :-)

We were able to successfully point our website to our own SQL Server.  However, we were unable to migrate the data from the Yessql file into our new database.  So, we created a new site and began adding the content back (not a super big deal) but for some reason we do not have the Content Part 'Autoroute' which we need to map pages.  Any idea why we do not see Autoroute any more?
